app/controllers/payment-links/review-controller.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+// NPM dependencies
+const lodash = require('lodash')
+
+// Local dependencies
+const {response} = require('../../utils/response.js')
+const paths = require('../../paths')
+
+module.exports = (req, res) => {
+  const pageData = lodash.get(req, 'session.pageData.createPaymentLink', {})
+
+  if (!pageData.paymentLinkTitle) {
```

Comment:
As discussed this check could probably for the next user story, at the time of submission just before creating the payment link.